content/22.vaccines.md

```diff
@@ -470,6 +469,76 @@ Similar to BNT162b2, the vaccine was associated with mild-to-moderate adverse ef
+Overall efficacy of mRNA vaccines was estimated at 91% with full vaccination, similar to the reports from the clinical trials.
+The occurrence of fevers was also lower in individuals who were partially or fully vaccinated, and the duration of symptoms was approximately 6 days shorter.
+<!--Flesh this out, what were the relative rates of strains?-->
+
+### Vaccines and Community Spread
+
+The vaccine clinical trial data demonstrate a significant reduction in the likelihood of contracting symptomatic COVID-19.
+However, given that vaccination rates are unlikely to reach 100%, another concern is whether vaccinated individuals with or without symptoms of COVID-19 can still spread SARS-CoV-2.
+This question is made up of several components.
+The crux is whether vaccinated individuals with a SARS-CoV-2 infection, regardless of symptom status, are as contagious as unvaccinated, infected individuals.
+Additionally, as outlined above, an important qualification is that the variants of SARS-CoV-2 circulating at the time of each study must be considered in light of the effect of evolution on vaccine efficacy.
+
+The phase 2/3 clinical trials evaluating the mRNA vaccines assessed vaccine efficacy based on COVID-19 diagnosis, thereby detecting only patients who received a diagnosis.
+In order to identify patients infected with SARS-CoV-2 who did not receive a diagnosis, for example, potentially those who did not develop symptoms, it would be necessary to conduct routine PCR testing even in the absence of symptoms.
+Prior to the development of vaccines, the evidence suggested that asymptomatic individuals could still spread SARS-CoV-2.
+Investigation of viral dynamics of asymptomatic infection in early 2020 indicated that asymptomatic patients continued to shed the virus for a duration similar to that of symptomatic patients [@doi:10.1016/j.ijid.2020.05.030] (although viral shedding should not be conflated with contagiousness without further investigation [@individual-pathogenesis]).
+Another study found viral load to be higher in the nasopharyngeal/oropharyngeal samples of asymptomatic patients compared to symptomatic patients hospitalized due to symptoms and/or known exposure [@doi:10.1007/s15010-020-01548-8].
+However, the sample size in both of these studies was small, and a larger study found higher viral load in symptomatic than asymptomatic cases [@doi:10.3390/v13020304] along with a systematic review finding a reduced probability of asymptomatic transmission [@doi:10.3138/jammi-2020-0030].
+While far from conclusive, these studies therefore suggest that asymptomatic cases still cary a risk of transmitting SARS-CoV-2.
+
+One important consideration is therefore how likely vaccinated individuals are to develop asymptomatic SARS-CoV-2.
+Considering asymptomatic cases is necessary to establish a more complete picture of efficacy with respect to spread.<!-- To Do: Possible figure indicating pyramid of risk with and without vaccines?-->
```

Comment:
```suggestion
Considering asymptomatic cases are necessary to establish a more complete picture of efficacy with respect to spread.<!-- To Do: Possible figure indicating pyramid of risk with and without vaccines?-->
```

Reply:
The figure is a nice thought there. Even some sort of infographic of current evidence. Important too now to consider pediatric vaccination. Maybe to include in its own discrete paragraph. I have just wrote up about that topic, so might try incorporate that in.

Reply:
Novavax and Astra Zeneca are all making strides too, but the most successful have been the two mRNA candidates.